client/src/ui/CourseReviews.tsx

```diff
@@ -19,10 +19,11 @@ export default function CourseReviews({
     <div>
       {reviews.map((review) => (
         <ReviewCard
-          key={review._id}
-          info={review}
+          reviewId={review._id}
+          review={review}
           reportHandler={reportReview}
           isPreview={false}
+          profilePage={false}
```

Comment:
since `isPreview ` is the name we are using for another field, maybe we can name this something similar like `isProfile` for coherence?